docs/dust.md

```
@@ -0,0 +1,71 @@
+        - name: dust.conf
+          contents: |
+            [Service]
+            ExecStartPre=/usr/bin/sh -c "readlink --canonicalize /etc/extensions/dust.raw > /tmp/dust"
+            ExecStartPre=/usr/lib/systemd/systemd-sysupdate -C dust update
+            ExecStartPost=/usr/bin/sh -c "readlink --canonicalize /etc/extensions/dust.raw > /tmp/dust-new"
+            ExecStartPost=/usr/bin/sh -c "if ! cmp --silent /tmp/dust /tmp/dust-new; then systemd-sysext refresh; fi"
+```
+
+## Building locally
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)